tests/builder_test.py

```diff
@@ -1131,5 +1131,40 @@ def test_to_ufo_draw_paths_qcurve(self):
         self.assertEqual(first_segment_type, 'qcurve')
 
 
+class GlyphPropertiesTest(unittest.TestCase):
```

Comment:
you're testing `to_ufos` function (or that what you are calling in the test), so this `test_glyph_color` method should go to `ToUfosTest` class, I think.

@belluzj wdyt?

Reply:
To me the current class name is good because it spells out the intent of the tests (test glyph properties) rather than the technical details (the test calls to_ufos). Beyond that, I don't know what the best practice is for organizing tests in python...